pkg/testutil/test/command.go

```diff
@@ -117,7 +117,10 @@ func (gc *GenericCommand) Run(expect *Expected) {
 		// Build the debug string - additionally attach the env (which iCmd does not do)
 		debug := result.String() + "Env:\n" + strings.Join(env, "\n")
 		// ExitCode goes first
-		if expect.ExitCode == -1 {
+		if expect.ExitCode == -2 { //nolint:revive
+			// -2 means we do not care at all about exit code
+		} else if expect.ExitCode == -1 {
+			// -1 means any error
```

Comment:
These magic numbers should be defined as `const`

Reply:
Yep, good point.

Done.